<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <widget class="QPushButton" name="toDataWindowButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>数据处理</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="showing">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
